src/tests/miscellaneous/mock-adb/setup-mock-adb.d.ts

```diff
@@ -22,3 +22,7 @@ export async function setupMockAdb(config: MockAdbConfig): Promise<void>;
 
 export type CommonAdbConfigName = 'single-device';
```

Comment:
we should update this too in this or a future PR, looks like we are missing some

Reply:
Fixed